### PR TITLE
bext: fix firefox release script

### PR DESCRIPTION
**Pull request by taras-yemets**
Fixes the Sourcegraph browser extension Firefox release script.
The previous script implementation worked fine with `yarn` v1 but broke after upgrading to v3.

Runs:
- [failed](https://buildkite.com/sourcegraph/sourcegraph/builds/172544#01833c40-b88a-4962-9174-078e9b1fe94f/152-246) (`./scripts/release-firefox.sh: line 11: web-ext: command not found`)
- [passing](https://buildkite.com/sourcegraph/sourcegraph/builds/172573#01833cc7-c27b-4f0c-9f24-09118d57b3ce) (`WebExtError: The extension could not be signed` resulting in `Error: The command exited with status 1` is expected). 

## Test plan
- Browser extension release process passing in CI (as in example above)

## App preview:

- [Web](https://sg-web-taras-yemets-fix-bext-release-for.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-iuywthnrgr.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Review by philipp-spiess — APPROVED** [review]
🤷

File: client/browser/scripts/release-firefox.sh [hunk 1]
```diff
@@ -8,7 +8,8 @@ rm -rf build/web-ext
 mkdir -p build/web-ext
 
 # Sign the bundle
-web-ext sign --source-dir ./build/firefox --artifacts-dir ./build/web-ext --api-key "$FIREFOX_AMO_ISSUER" --api-secret "$FIREFOX_AMO_SECRET"
+yarn add web-ext
```
Comment (taras-yemets) [inline client/browser/scripts/release-firefox.sh:11]: @philipp-spiess, do you have better ideas than installing `web-ext` here?
Adding it to the project's dev dependencies (https://github.com/sourcegraph/sourcegraph/pull/41655/commits/479fbc9817b5fc53abc6de25b5f849df14e84269) results in `Usage Error: Couldn't find a script named "web-ext"` ([buildkite](https://buildkite.com/sourcegraph/sourcegraph/builds/172569#01833cbd-d942-4998-a06f-8e3853b70b2d/152-220)).

Comment (taras-yemets) [reply client/browser/scripts/release-firefox.sh:11]: By the way, do you have an idea why it was failing?

Comment (philipp-spiess) [reply client/browser/scripts/release-firefox.sh:11]: Hmm maybe it's just not available in the global `PATH`? You could try if this works instead:

```suggestion
./node_modules/.bin/web-ext sign --source-dir ./build/firefox --artifacts-dir ./build/web-ext --api-key "$FIREFOX_AMO_ISSUER" --api-secret "$FIREFOX_AMO_SECRET"
```

Comment (taras-yemets) [reply client/browser/scripts/release-firefox.sh]: Unfortunately, it still errors out
```
./scripts/release-firefox.sh: line 11: ./node_modules/.bin/web-ext: No such file or directory
```

**Pull request merged**